README.md

```diff
@@ -132,7 +138,8 @@ Success. Exiting
 
 * Make sure the fields `nsapi.default-cellular-sim-pin`, `nsapi.default-cellular-plmn`, `nsapi.default-cellular-apn`, `nsapi.default-cellular-username` and `nsapi.default-cellular-password` from the `mbed_app.json` file are filled in correctly. The correct values should appear in the user manual of the board if using eSIM or in the details of the SIM card if using normal SIM.
 * Enable trace flag to have access to debug information `"mbed-trace.enable": true` and `"cellular.debug-at": true`.
-* Check that a default modem is defined, e.g. `"GENERIC_AT3GPP.provide-default": true`, `"GENERIC_AT3GPP.tx": "<tx-pinmap>"`, ...
+* Error Message: Assertion failed: iface usually means that a default modem is not defined, e.g. `"GENERIC_AT3GPP.provide-default": true`
+* If the modem does not respond to (AT) queries, check that UART pins (tx, rx, rts, cts) are connected and defined, e.g. `"GENERIC_AT3GPP.tx": "<tx-pinmap>"`, ...
```

Comment:
Could you also add comment about powering the modem, something like "Make sure modem is properly powered. Some modems requires external power supply.".